app/uk/gov/hmrc/merchandiseinbaggage/repositories/DeclarationJourneyRepository.scala

```diff
@@ -28,22 +28,23 @@ import uk.gov.hmrc.merchandiseinbaggage.model.core.DeclarationJourney.{format, i
 import uk.gov.hmrc.mongo.MongoComponent
 import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 
+import java.time.Instant
 import java.util.concurrent.TimeUnit
 import javax.inject.{Inject, Named, Singleton}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
 class DeclarationJourneyRepository @Inject()(mongo: MongoComponent, @Named("declarationJourneyTimeToLiveInSeconds") ttlInSeconds: Int)
```

Comment:
instead of injecting property here please inject AppConfig and use property from application.conf